### ISSUE TITLE
Implement NavigatorLanguage::languages

### ISSUE THREAD

**Issue by Ms2ger** [servo/servo#10073]
Depends on #10072. (This implies we will not fix this issue in the near future.)
- Spec: https://html.spec.whatwg.org/multipage/#navigatorlanguage.
- Code: `components/script/dom/webidls/Navigator.webidl`, `components/script/dom/navigator.rs`, `components/script/dom/webidls/WorkerNavigator.webidl`, `components/script/dom/workernavigator.rs`.
- Tests: to be written, as far as I know.

**Comment by craftytrickster** [comment]
Do you mind if I take this?

**Comment by jdm** [comment]
@craftytrickster See the first sentence of the original comment.

**Comment by craftytrickster** [comment]
@jdm I'm not familiar with your use of webidl, but can't this at least be partially implemented with a Vector?

**Comment by jdm** [comment]
For #10072 specifically, the main reason to hold off implementing it is that Gecko has not and the FrozenArray feature is really new.

**Comment by craftytrickster** [comment]
Thanks for the info, nevermind me taking this then :)

**Comment by nox** [comment]
This has yet to be implemented.

**Comment by nox** [comment]
And it's blocked by #10072.

**Comment by jdm** [comment]
Closing in favour of #27210. We don't support frozen arrays in our codegen yet, but we can manually return a frozen array.
